test/functional/api/window_spec.lua

```diff
@@ -551,6 +551,33 @@ describe('API/win', function()
     end)
   end)
 
+  describe('reset dir', function()
+    it('reset dir in nvim_open_win', function()
```

Comment:
should this test live in the nvim_open_win describe() block? : https://github.com/neovim/neovim/blob/57adf8c6e01d9395eb52fe03571c535571efdc4b/test/functional/api/window_spec.lua#L1164